docs/install.rst

```diff
@@ -234,11 +234,10 @@ Installing from source
 Installation from source is only supported for developers and the
 whole process is described in the :doc:`contributing`.
 
-.. warning:: Please do **not** use ``python setup.py install`` or
-   ``python pip install .``. Please do **not** attempt the
-   installation commands as superuser/root and/or without virtual
-   environment, e.g. ``sudo python setup.py install``, ``sudo pip
-   install``, ``sudo ./venv/bin/...``. These modes of operation might
-   corrupt your operating system and are **not supported** by the
-   Certbot team!
+.. warning:: Please do **not** use ``python setup.py install``, ``python pip
+   install .``, or ``easy_install .``. Please do **not** attempt the
```

Comment:
Very sure. It works correctly when building the docs locally and [this section already has double backticks across multiple lines](https://github.com/certbot/certbot/pull/4960/files?diff=unified#diff-62c330cd8da31d476bb63b51bb8b1307L240) in `master` which is [displayed properly](https://certbot.eff.org/docs/install.html#installing-from-source).